ecosystem/indexer-server/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16 as builder
+ARG PATH
+
+WORKDIR /usr
+
+RUN echo "copy indexer server..."
+COPY . .
```

Comment:
avoid copying over the entire directory -- just the files that are required for the build

Reply:
only copy some  source files required for the build